src/library/scala/util/Random.scala

```diff
@@ -14,6 +14,7 @@ import scala.collection.mutable.ArrayBuffer
 import scala.collection.BuildFrom
 import scala.collection.immutable.{ List, LazyList }
 import scala.language.{implicitConversions, higherKinds}
+import scala.util.chainingOps._
```

Comment:
https://travis-ci.org/scala/scala/jobs/415576743#L551

```
[error] /home/travis/build/scala/scala/src/library/scala/util/Random.scala:17: object chainingOps is not a member of package util
[error] import scala.util.chainingOps._
[error]                   ^
[error] /home/travis/build/scala/scala/src/library/scala/util/Random.scala:44: value tap is not a member of Array[Byte]
[error]   def nextBytes(n: Int): Array[Byte] = Array.ofDim[Byte](n).tap(self.nextBytes)
[error]                                                             ^
```

Reply:
should be `scala.util.chaining._`